Documentation/subcommands/fly.md

```diff
@@ -0,0 +1,6 @@
+# rkt fly
+
+***This subcommand does not exist yet.***
+It will be a synonym for `rkt run --stage1=/path/to/stage1-fly.aci`.
```

Comment:
I actually don't want this to be synonymous, because then `rkt run` is NOT a consistent interface. I'd rather have the `fly` stage1 not actually be a true stage1. For example, it might omit a particular annotation, and then `rkt run` would bail out if it's passed as a stage1; whereas `rkt fly` would have different expectations